lib/build-frontend.js

```diff
@@ -109,8 +108,8 @@ function isNpmInstallNeeded(){
         let packageInfo = JSON.parse(fs.readFileSync(packageJsonFilePath, 'utf8'))
         if(packageInfo){
 
-            let hasDependency = (packageInfo.dependencies &&  ! _.isEmpty(packageInfo.dependencies)) ||
-                                (packageInfo.devDependencies &&  !_.isEmpty(packageInfo.devDependencies))
+            let hasDependency = (packageInfo.dependencies &&  Object.keys(packageInfo.dependencies).length>0) ||
```

Comment:
If package.dependencies is undefined, Object.keys() will error out. Maybe have an undefined check for packageInfo.dependencies? _.keys(undefined) returns [] though and doesn't error out.

Reply:
if the package.dependencies is not defined, then the second operation of the && operand will not be evaluated.